src/main/java/org/reaktivity/specification/nukleus/CoreFunctions.java

```diff
@@ -81,6 +82,27 @@ public static Random random()
         return array;
     }
 
+    @Function
+    public static byte capabilities(
+        String capability,
+        String... optionalCapabilities)
+    {
+        return of(capability, optionalCapabilities);
+    }
+
+    private static byte of(
+        String name,
+        String... optionalNames)
+    {
+        byte capabilityMask = 0x00;
```

Comment:
Recommend we `assert` that `optionalNames` cannot cause us to run out of bit positions in the `byte`.